plantisc2/tasks/db.yml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
For this role you suggested me to check something in Galaxy, I'm havein a look at [this](https://galaxy.ansible.com/geerlingguy/mysql/)